dependencies.props

```diff
@@ -13,7 +13,7 @@
   <!-- Full package version strings that are used in other parts of the build. -->
   <PropertyGroup>
     <CoreClrPackageVersion>1.1.3-servicing-25416-01</CoreClrPackageVersion>
-    <XunitPackageVersion>2.1.0</XunitPackageVersion>
+    <XunitPackageVersion>2.2.0-beta2-build3300</XunitPackageVersion>
```

Comment:
Have you tried running the functional tests with this change?  I could see this being a problem.